src/lib/kdump/kdump_calibrator.rb

```diff
@@ -131,13 +162,16 @@ def system
     def run_kdumptool
       out = Yast::SCR.Execute(Yast::Path.new(".target.bash_output"), kdumptool_cmd)
       if out["exit"].zero?
+        # ensure that fadump is set to unsupported as fadump can be missing in output
+        # and it means fadump not supported ( as oppose to high mem which is always in output)
```

Comment:
typo? _opposed_

Reply:
NP: Anyway, the comment is a bit difficult to understand to me. What about something like


> If fadump is missing from output, it means it isn't supported (unlike high mem, which is always in output). Thus, let's set fadump related values to zero before parsing the kdumptool output